scenario_files/pilot1-full-test-replay.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0.0" encoding="UTF-8"?>
+<scenario>
+  <identifier>
+    <name>Pilot 1 Network Test Scenario</name>
+  </identifier>
+  <OpenDrive>
+    <MapFileName name="Town04.xodr">
+      <MapOrigin latitudeInDegrees="0.0" longitudeInDegrees="0.0" heightAboveEllipsoidInMeters="0"> </MapOrigin>
+    </MapFileName>
+  </OpenDrive>
+
+  <vehicles>
+
+    <vehicle name="NISSAN-SVM" bsmid="f03ad614" type="vehicle.nissan.micra" owner="CARLA-NISSAN" material="Aluminum" lvcIndicator="Live">
+      <!-- <userData><CARLA manual="true" generate_bsms="false" ></CARLA></userData> -->
```

Comment:
Why is this commented out?

Reply:
this is what you have do to when you do a replay. you dont want it to be a manual vehicle because it will spawn with physics. you just want it to spawn as a dummy vehicle with no physics so the updates will move it around. in this case, removing this makes it technically a "live" vehicle.